biggus/__init__.py

```diff
@@ -490,12 +490,16 @@ def __array__(self, dtype=None):
         return np.asarray(result, dtype=dtype)
 
     def __str__(self):
-        fmt = '<Array shape={} dtype={!r} size={}>'
-        return fmt.format(self.shape, self.dtype, size(self))
+        fmt = '<Array shape=({},) dtype={!r} size={}>'
```

Comment:
Regarding https://github.com/SciTools/biggus/pull/139/files#r27195913, perhaps just remove the trailing ``,``?